test/test_jit.py

```diff
@@ -1457,7 +1457,10 @@ def test_pattern_based_rewrite(self):
 graph(%a, %b, %c):
   %q = aten::mul(%a, %b)
```

Comment:
Can we have a test for the special case outlined in the comments? What about the corner case when we want to replace the entire graph with an empty graph?

Reply:
> Can we have a test for the special case outlined in the comments? 

What case are you referring to? We currently have tests for: 1) fusion (replace many with one), 2) combining (replace many with many), 3) removal (replace something with nothing).

> What about the corner case when we want to replace the entire graph with an empty graph?

We would be able to do that if we add support for wildcards. Currently there is no way to do it (but I also have a question, why do you want do this:) ).

Reply:
I agree, this is only a corner case. I think I missed seeing the test case for removal of nodes, so that is sufficient.